mappings/servers/hvbc/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id": "hvbc",
+    "name": "HVBC",
+    "description": "Practice Server For Legit player",
```

Comment:
If you are concerned about proper grammar (which we do care to a degree), I recommend that you adjust the description.

A more proper description could be:
- “A practice server for legit players”
- “A practice server for legitimate players”

You don’t need to be restricted to these two options, but these should suffice while keeping the main description value.

If you want to change the description entirely, feel free to do so